src/screens/PoolTogether/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "PoolTogetherDash.js",
```

Comment:
PoolTogetherDashboard? Honestly I didn't get in the first place what this stands for.